collection-scripts/gather_etcd

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+ETCD_LOG_PATH="${BASE_COLLECTION_PATH}/etcd_info"
+
+RUNNING_ETCD_POD=$(oc get pod -n openshift-etcd --no-headers|grep Running| grep -o -m 1 '\S*etcd\S*')
+
+function ocp4etcdctl {
+    oc -n openshift-etcd exec -c etcdctl ${RUNNING_ETCD_POD} -- etcdctl $@
+}
+
+if [[ -z $RUNNING_ETCD_POD ]];then
+    echo "ERROR: No running etcd pods found" >&2
+    exit 1
+fi
+
+mkdir -p ${ETCD_LOG_PATH}
+
+echo "INFO: Starting getting etcd information"
```

Comment:
Should this print the name of `RUNNING_ETCD_POD`?

Reply:
Good idea! I'll implement that.

Reply:
This still doesn't include `RUNNING_ETCD_POD`.  Did you forget to commit and/or push?

Reply:
It was already added but in another log statement below. Please see: https://github.com/palonsoro/must-gather/blob/gather_etcd/collection-scripts/gather_etcd#L22

Reply:
Yea I still don't see where we are 'printing' the pod name. Can we add that here or to the items you reference.

Reply:
Hello, sorry the line number in my comment above is no longer correct due to changes happened after it.

Please look now at at: https://github.com/palonsoro/must-gather/blob/gather_etcd/collection-scripts/gather_etcd#L27